server/test/mocks/mock_lib_wrapper.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef INS_SERVER_TEST_MOCKS_LIB_WRAPPER_HPP
+#define INS_SERVER_TEST_MOCKS_LIB_WRAPPER_HPP
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "lib_wrapper.hpp"
+
+using namespace Pistache::Http;
```

Comment:
Don't use `using` in the headers. :smiling_imp:

Reply:
Done